mrmustard/physics/representations.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Xanadu Quantum Technologies Inc.
+# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Comment:
in general we should leave the earlier date, so that the copyright is covered from when the original file was created